src/main/java/nl/novi/techiteasy1121/models/Television.java

```diff
@@ -1,217 +1,230 @@
 package nl.novi.techiteasy1121.models;
 
-import jakarta.persistence.*;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 
 @Entity
+@Table(name = "televisions", schema = "public") // Ensure the schema is set to "public"
 public class Television {
 
-    //  Een entiteit moet een primary key bevatten(id)
     @Id
-    // GeneratedValue betekend dat je deze waarde niet zelf hoeft in te vullen, dit doet Spring Boot voor jou bij het opslaan in de database.
-    @GeneratedValue
-    Long id;
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
 
-    //  variable declaraties
-    private String type;
-    private String brand;
+    @Column(name = "name")
     private String name;
-    private Double price;
-    private Double availableSize;
-    private Double refreshRate;
+
+    @Column(name = "brand")
+    private String brand;
+
+    @Column(name = "price")
+    private double price;
+
+    @Column(name = "original_stock")
```

Comment:
Ook hier is de columnnam al automatisch "original_stock". JPA vertaalt camelCase naar snake_case, omdat sql geen hoofdletters kent.